nix/packagesFromDirectory.nix

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
I think there's still the old version of this function in `nix/overlays/lib.nix`, which you can probably delete now

Reply:
That's in `mercury-web-backend`!

Reply:
Oh, whoops!  I didn't notice this was a different repo 😅.  Never mind me, then.